docker/opensearch/init-opensearch.sh

```diff
@@ -41,7 +31,7 @@ else
               "keyword": {"type": "keyword", "ignore_above": 256},
               "exact": {
                 "type": "text",
-                "analyzer": "exact_case_sensitive_analyzer"
+                "analyzer": "standard"
```

Comment:
I think case sensitivity is overkill for now. But, now that I think about it, maybe it should be another field and we should us it to rank the case sensitive matches higher.